cattle/plugins/volmgr/volmgr.py

```diff
@@ -203,6 +203,20 @@ def update_managed_volume(instance, config, start_config):
     start_config['binds'] = new_binds_map
 
 
+def snapshotExists(snapshot_uuid, volume_uuid):
+    volumes = v.list_volumes(volume_uuid)
```

Comment:
No, it would search for that specific volume. But if you have tons of snapshots, it would be slower.